src/main/java/com/iexec/common/chain/DealParams.java

```diff
@@ -121,6 +121,9 @@ public class DealParams {
     @JsonProperty("iexec_tee_post_compute_fingerprint")
     private String iexecTeePostComputeFingerprint;
 
+    @JsonProperty("iexec_sms_url")
```

Comment:
Should we use a "default" sms if the url is not specified in the deal?

Reply:
It would also reduce gas cost because I think most of the time we would like to use the default SMS.

Reply:
I don't think we would like to use a default SMS as of now. It may be provided by our SMS catalog in the future, depending on the TEE tag. We'll have to discuss this with @PierreJeanjacquot though.
I propose we stay with no default SMS for now and we'll change it depending on our discussion. Is this OK for you?

Reply:
No problem yes.